app/helpers/patients_helper.rb

```diff
@@ -37,7 +39,11 @@ def voicemail_options(current_value = nil)
         [t('dashboard.helpers.voicemail_options.no'), 'no'],
         [t('dashboard.helpers.voicemail_options.yes'), 'yes'],
     ]
-    full_set = standard_options + Config.find_or_create_by(config_key: 'voicemail').options
+    full_set = Config.find_or_create_by(config_key: 'voicemail').options
```

Comment:
This also gets a little weird when the defaults aren't available, can we have this one also exempt from the hide default options rule?